jetty-ee10/jetty-ee10-webapp/src/test/java/org/eclipse/jetty/ee10/webapp/HugeResourceTest.java

```diff
@@ -603,8 +603,9 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
                     IO.copy(inputStream, byteCounting);
                     out.printf("part[%s].inputStream.length=%d%n", part.getName(), byteCounting.getCount());
                 }
-                catch (IOException e)
+                catch (Throwable e)
                 {
+                    resp.setStatus(500);
```

Comment:
Can't set 500 in the majority of cases here, the response is already committed.
Probably better to let the exception just flow out.

Reply:
Okay, rethrowing as `AssertionError` then.